Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.20.7 as builder
+
+WORKDIR /go/mx-chain-ws-connector-firehose-go
+COPY . .
+RUN go mod tidy
+# Multiversx node
```

Comment:
udpate or remove this comment since it is not related to mx-chain-go node

Reply:
done.